app/controllers/solidus_paypal_commerce_platform/orders_controller.rb

```diff
@@ -2,12 +2,12 @@
 
 module SolidusPaypalCommercePlatform
   class OrdersController < ::Spree::Api::BaseController
-    before_action :load_order
-    before_action :load_payment_method, only: :create
+    before_action :load_order, only: :update_address
+    before_action :load_payment_method, only: :show
     skip_before_action :authenticate_user
 
-    def create
-      authorize! :update, @order, order_token
+    def show
+      authorize! :show, @order, order_token
```

Comment:
I'm a bit concerned by this change. Having had a look at the next commit feels more like the name conflict is an indication that we're dealing with two different resources (in REST terms) and that we probably should add a second controller.

Of course I might be missing some important point 😅, wdyt about the additional controller?

Reply:
I wasn't 100% sure about this, but my thought process was: The new `create` method actually creates an order on Solidus, so `order#create` makes sense. The old method (renamed to `show`) `shows` the order to PayPal, so `order#show` also makes sense... kinda. 

The new `show` method IMO should not have been `create` in either case, since it doesn't create anything :P 

Off the top of my head, maybe it would make more sense if the method that `shows` the order to PayPal is instead in `paypal_orders_controller` - since it's `show`ing PayPal a `paypal_order`? What do you think?

Reply:
Yeah, I guess I was mislead by the create_order api call, I think it's fine as long as each call to show isn't creating a new paypal-order. The rename to _paypal_orders_ would certainly make the subject of the controller more explicit, so 👍👍👍

Reply:
Agreed - I moved the show action to the `paypal_orders_controller` for clarity. Thanks!